.github/workflows/Build.yaml

```diff
@@ -101,19 +114,22 @@ jobs:
           commit_message: "🤖 Updates screenshots"
 
       # Run local tests after screenshot tests to avoid wrong UP-TO-DATE. TODO: Ignore screenshots.
-      - name: Run local tests and create report
-        if: always()
+      - name: Run local tests
         run: ./gradlew testDemoDebug :lint:test
-      # Replace task exclusions with `-Pandroidx.baselineprofile.skipgeneration` when
-      # https://android-review.googlesource.com/c/platform/frameworks/support/+/2602790 landed in a
-      # release build
+
+      - name: Setup GMD
```

Comment:
Is setting up GMD necessary in the `Build` workflow?

Reply:
Yes. That's the part where the [GH Action](https://github.com/android/nowinandroid/actions?query=branch%3Abw%2FbpGenOnKvm) failed consistently.

Reply:
This is a workaround for GHA. Gradle runs tasks in parallel up to the number specified by the `--max-workers` flag. The GHA runner we're using doesn't have enough processing power to run an emulator while other build tasks, like the Kotlin compiler and aapt, are running. This causes the emulator process to crash. By running the setup task in a separate step, we can ensure that it runs while no other tasks are running in parallel. Alternatively, we could upgrade the GHA runner to one with more cores.